ccdproc/image_collection.py

```diff
@@ -57,11 +62,15 @@ class ImageFileCollection(object):
         Raised if keywords are set to a combination of '*' and any other
         value.
     """
-    def __init__(self, location=None, keywords=None, info_file=None):
+    def __init__(self, location=None, filenames=None, keywords=None,
```

Comment:
New arguments should be added at the end of the list of arguments to ensure backwards-compatibility.